### DISCUSSION TITLE
Problem on installation

### DISCUSSION THREAD

**Discussion by j03-dev** [tsirysndr/rockbox-zig#75] [Q&A] [answered]
![Screenshot From 2024-12-25 23-07-50](https://github.com/user-attachments/assets/56dbda29-90f8-4ef7-8447-c3ae9fa86b66)
I'm actually on Fedora 41, Intel 
I have this actually problem when I start the server.

**Comment by tsirysndr** [comment]
Thank you for reporting the issue.

This does not appear to be a bug: https://github.com/ziglang/zig/wiki/FAQ#why-do-i-get-illegal-instruction-when-using-with-zig-cc-to-build-c-code

Could you provide more detailed information about your Intel CPU? The program works correctly on modern Intel CPUs.

It’s possible that we may need to disable certain Zig compiler optimizations.

As a temporary solution, you can try recompiling `rockboxd` from source on your local machine.

**Reply by j03-dev** [reply]
I tried to do that, but it failed. definedCMacro is deprecated. I tried adding -D to the CFLAGS, like -DZIG_APP

Maybe you should specify which version of Zig you're using to build this project.

**Reply by tsirysndr** [reply]
could you provide more detailed information about your command and your CPU?

**Reply by j03-dev** [reply]
![image](https://github.com/user-attachments/assets/9113ff26-1be5-44b4-bdcf-d1b8d94ae290)

**Reply by tsirysndr** [reply]
Could you please follow the correct commands as described in the README?

**Reply by j03-dev** [reply]
This is the command that I used , I only change the first command to install deps!
![Screenshot From 2024-12-26 10-52-59](https://github.com/user-attachments/assets/d3f45c65-b126-4d38-9b25-7938ce269f6e)
![image](https://github.com/user-attachments/assets/19e5029f-c6f9-4ef4-901b-916f12a6bb04)

**Reply by tsirysndr** [reply]
could you provide your zig version (zig 0.13.0)?

**Reply by j03-dev** [reply, marked as answer]
Nope! The master 0.14.0-dev.2563+af5e73172, I'll change it, to 0.13.x
